src/transformers/models/sam/convert_sam_to_hf.py

```diff
@@ -173,38 +173,38 @@ def convert_sam_checkpoint(model_name, checkpoint_path, pytorch_dump_folder, pus
 
     elif model_name == "sam_vit_h_4b8939":
         inputs = processor(
-            images=np.array(raw_image), input_points=input_points, input_labels=input_labels, return_tensors="pt"
+            images=np.array(raw_image), input_points=[input_points], input_labels=input_labels, return_tensors="pt"
         ).to(device)
 
         with torch.no_grad():
             output = hf_model(**inputs)
         scores = output.iou_scores.squeeze()
 
-        assert scores[-1].item() == 0.9712603092193604
+        assert scores[-1].item() == 0.9833242893218994
```

Comment:
overall let's remove this it does not make sense to tests stuff when you convert as there can be many models

Reply:
will remove the asserts